rudder-lang/tests/helpers/cis9.1_cron.rl

```diff
@@ -0,0 +1,27 @@
+# Generated from json technique
+@format=0
+@name="CIS 9.1 configure cron"
+@description="CIS benchmark section for cron."
+@version="1.0"
+@parameters= []
+
+resource CIS_9_1_configure_cron()
+
+CIS_9_1_configure_cron state technique() {
+  MethodCall
+  MethodCall
+  MethodCall
+  MethodCall
```

Comment:
Is this normal?

Reply:
oh, it was a test I did before implementing MethodCalls :). Don't mind it, it is not a file reference, just the output of a test, it will be overwritten in next related PR